spec/system/hiring_staff_can_manage_vacancy_applications_spec.rb

```diff
@@ -0,0 +1,218 @@
+require "rails_helper"
+
+RSpec.describe "Hiring staff can manage vacancy applications" do
```

Comment:
I would rather we said `Publishers can manage job applications for a vacancy` for two reasons
- Publishers is the name of the model 
- Vacancy applications makes me want to cry 😂

Reply:
Also, this seems like an awful lot of examples when we want system specs to be happy path